starcheck/calc_ccd_temps.py

```diff
@@ -61,10 +60,28 @@
 except Exception:
     VERSION = 'dev'
 
+CM_ACA_MODEL_DATA, CM_ACA_MODEL_INFO = (
+    chandra_models.get_data('chandra_models/xija/aca/aca_spec.json'))
```

Comment:
Best to avoid "heavy" functions that might fail in the import. I've become of fan of `lru_cache` for stuff like this. These can be written without the `get_` in front because conceptually they are more like your original global constants.
```
@functools.lru_cache()
def aca_model_and_info():
    model, info = chandra_models.get_data('chandra_models/xija/aca/aca_spec.json')
    return model, info

@functools.lru_cache()
def aca_model_spec():
    model, _ = aca_model_and_info()
    return json.loads(model)

@functools.lru_cache()
def chandra_models_version():
    _, info = aca_model_and_info()
    return info["version"]
```

Reply:
Sure.  I generally avoid "heavy" functions on import but did consider this in context (calc_ccd_temps only used by starcheck and we have no alternate strategy in mind if getting the model doesn't work).  But if the lazy imports are still considered nicer given context I'll update to that before testing.